src/storage-controller/src/lib.rs

```diff
@@ -1775,7 +1775,7 @@ where
                 let drop_notif = match description.data_source {
                     DataSource::Other(DataSourceOther::TableWrites) if read_frontier.is_empty() => {
                         pending_collection_drops.push(id);
-                        Some(self.persist_table_worker.drop_handle(id))
+                        Some(self.persist_table_worker.drop_handles(vec![id]))
```

Comment:
I tried in vain for longer than I care to admit to refactor this code so that we could call `self.persist_table_worker.drop_handles` once for all tables in `self.pending_compaction_commands`. I figured that what we have is good enough for now and I'll refactor this function in another follow up PR.

Reply:
Just slap a `TODO` on it and refactor later!